### ISSUE TITLE
A11y_AzureDataStudio_ServiceSettings_AI4D:Incorrect name property provided for the "SQL Server Master" combo box."

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7608]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.1.2;#WCAG4.2.1;#DesktopApp;#NVDA;#JAWS;#Win32;#AI4D;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name : AI4D

**Repro Steps:**

1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button.
8.	Open AI4D and check the name property for the "SQL Server Master" combo box.

**Actual:**
Incorrect name property provided for the "SQL Server Master" combo box.

**Expected:**
Name property should provided for the  combo box as "SQL Server Master" and screen reader should announces the same.

**User Impact:**
Screen reader user does not understand regarding the name of the control.

**Attachment for Reference:**
[7608_A11y_AzureDataStudio_ServiceSettings_AI4D.pptx](https://github.com/microsoft/azuredatastudio/files/3712455/7608_A11y_AzureDataStudio_ServiceSettings_AI4D.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Data Studio
Version: 1.15.0-insider (user setup)
 I am not able to found the  "Hd availability  settings"  control.
Please confirm  that whether the control is removed ?
If control is removed the issue is no longer to reproduces.

**Comment by Charles-Gagnon** [comment]
@vinodbk197 This page has indeed been refactored since this issue was opened. The dropdown is still there though - it's just under the general *Scale Settings* section now. (@alanrenmsft to double-confirm) I confirmed that it has the expected name property 

![image](https://user-images.githubusercontent.com/28519865/72393934-3f917a80-36e9-11ea-958d-cd1862c6c267.png)

Let us know if this matches your expectations.

**Comment by vinodbk197** [comment]
Verified the issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is resolved,Hence closed this issue.
Please refer the below attachment:
![image](https://user-images.githubusercontent.com/55909580/72731060-e201a000-3bb8-11ea-9e73-78aa49cb412c.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
